realtor/src/main/webapp/WEB-INF/jsp/editbuilding.jsp

```diff
@@ -0,0 +1,34 @@
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Edit building</title>
+</head>
+<body>
+<div align="center"><h2 style="color: slateblue">Add new building</h2></div>
+<div align="center">
+    <form:form action="/editbuilding?building.id=${building.id}" method="post" modelAttribute="building">
```

Comment:
`uilding.id=${building.id}` shouldn't be here, it should be as a `hidden` input in the form